velox/dwio/dwrf/reader/SelectiveStringDirectColumnReader.cpp

```diff
@@ -237,8 +237,8 @@ inline bool SelectiveStringDirectColumnReader::try8Consecutive(
   const char* data = bufferStart_ + start + bytesToSkip_;
   if constexpr (!sparse) {
     auto* lengths = rawLengths_ + rows[row];
-    uint16_t offsets[9];
-    bool gt4;
+    uint16_t offsets[9] = {0};
```

Comment:
These are left uninitialized for performance reason.  Can we suppress the warning for low level code like this?

Reply:
Yes, we can suppress warnings for low-level code like this.  I will remove this change.